python/py-spaghetti/Portfile

```diff
@@ -20,16 +20,14 @@ long_description    The PySAL {*}${description} package is an open-source Python
 
 homepage            https://pysal.org/spaghetti/
 
-checksums           rmd160  092dd6c9e7abc9fcba7b03044a11124d828c6c9c \
-                    sha256  9898ea86744b6ced9923f8bc213964045615f31ce14f304d20db0aecca40728d \
-                    size    22151842
+checksums           rmd160  ab48d4d070edd4274c22c2151cb33a7a96f9b0ef \
+                    sha256  9acf19e3b4bf72f83604bc1d00113712cbb7db7bd94984883f5823a6fb1da607 \
+                    size    22156820
 
 python.versions     39 310 311
```

Comment:
It won't work with 3.12?

Reply:
In it self probably, re dependencies I don’t know. See https://github.com/macports/macports-ports/pull/22049#issuecomment-1878671038 for earlier attempt for the pysal family of modules. I’ll return for a common approach for them all.